README.md

```diff
@@ -35,6 +30,15 @@ npm install
 ## Running Tests
 - npm test to run jest unit test suite for all components 
 
+## CRUD
+
+| Command | Method | Endpoint | Body | Purpose |
+| ------- | ------ | -------- | ---- | ------- |
+| Create | POST | '/restaurants/:restaurantName/:restaurantID/menu' | JSON | Add new restaurant to platform |
+| Read | GET | '/restaurants/:restaurantName/:restaurantID/menu' | none | Get data for given restaurant (defined in endpoint) |
+| Update | PUT | '/restaurants/:restaurantName/:restaurantID/menu' | JSON | Update restaurants information |
+| Delete | DELETE | '/restaurants/:restaurantName/:restaurantID/menu' | none | Delete restaurant from platform |
```

Comment:
Why are you using both "restaurantName" and "restaurantID"? It seems like either one should be sufficient to identify the item in the database.

Reply:
Hi Jordan, 

The learn documents ask us to be able to query with the item_name and item_id. Although the current queries don't make use of the name, I might use them later on. 

Thanks for the input,

Alex